ddtrace/contrib/requests/__init__.py

```diff
@@ -15,6 +15,15 @@
 
     session = TracedSession()
     session.get("http://www.datadog.com")
+
+To enable distributed tracing, for example if you call, from requests, a web service
+which is also instrumented and want to have traces including both client and server sides::
+
+    from ddtrace.contrib.requests import TracedSession
+
+    session = TracedSession()
+    session.distributed_tracing_enabled = True
```

Comment:
All our other contribs are using simply `distributed_tracing` as the option key.
Should we just do the same?

Reply:
Oh indeed, good catch. `aiohttp` https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/contrib/aiohttp/middlewares.py#L29 is the only one to use the `_enabled` flavor (and as it shows up 1st in alphabetical order, might explain why I used this...). Yes, I can rename it to `distributed_tracing` alone, we might fix `aiohttp` later if needed.